hack/common.sh

```diff
@@ -86,6 +85,7 @@ METRICS_EXPORTER_FULL_IMAGE_NAME="${METRICS_EXPORTER_FULL_IMAGE_NAME:-${DEFAULT_
 UX_BACKEND_OAUTH_FULL_IMAGE_NAME="${UX_BACKEND_OAUTH_FULL_IMAGE_NAME:-${DEFAULT_UX_BACKEND_OAUTH_FULL_IMAGE_NAME}}"
 
 NOOBAA_BUNDLE_FULL_IMAGE_NAME="quay.io/noobaa/noobaa-operator-bundle:master-20231217"
+ROOK_BUNDLE_FULL_IMAGE_NAME="docker.io/rook/rook-ceph-operator-bundle:master"
```

Comment:
Is rook planning to maintain this bundle image? We should not use the master tag will be continuously updated with the latest changes, we should use some particular tag to use some particular version image.

Reply:
Yes, that's what the plan is that they will be maintaining the rook downstream repo as well.
The master tag is just the placeholder for now, I will update it once we have the rook downstream changes merged.